src/multivariate.jl

```diff
@@ -154,40 +154,46 @@ end
 
 function Distributions._logpdf(d::TuringScalMvNormal, x::AbstractVector)
     σ2 = abs2(d.σ)
-    return -(length(x) * log(2π * σ2) + sum(abs2.(x .- d.m)) / σ2) / 2
+    return -(length(x) * log(twoπ * σ2) + sum(abs2.(x .- d.m)) / σ2) / 2
 end
 function Distributions.logpdf(d::TuringScalMvNormal, x::AbstractMatrix{<:Real})
     size(x, 1) == length(d) ||
         throw(DimensionMismatch("Inconsistent array dimensions."))
-    return -(size(x, 1) * log(2π * abs2(d.σ)) .+ vec(sum(abs2.((x .- d.m) ./ d.σ), dims=1))) ./ 2
+    return -(size(x, 1) * log(twoπ * abs2(d.σ)) .+ vec(sum(abs2.((x .- d.m) ./ d.σ), dims=1))) ./ 2
 end
 function Distributions.loglikelihood(d::TuringScalMvNormal, x::AbstractMatrix{<:Real})
     σ2 = abs2(d.σ)
-    return -(length(x) * log(2π * σ2) + sum(abs2.(x .- d.m)) / σ2) / 2
+    return -(length(x) * log(twoπ * σ2) + sum(abs2.(x .- d.m)) / σ2) / 2
 end
 
 function Distributions._logpdf(d::TuringDiagMvNormal, x::AbstractVector)
-    return -(length(x) * log(2π) + 2 * sum(log.(d.σ)) + sum(abs2.((x .- d.m) ./ d.σ))) / 2
+    s = sum(log, d.σ)
```

Comment:
sum(log....) to sum(log, ...)) which in the past caused sometimes problems with AD. Better not include such changes.